### ISSUE TITLE
遭遇Bug

### ISSUE THREAD

**Issue by MrGH-boss** [jungao1106/ICoT#15]
作者您好！我在复现您的论文时有一行代码报错我无法解决：**run.py->calculate_generated_text()->**inputs, sub_image_masks= processor(text = prompt, images=vision_x, padding=True, return_tensors="pt", return_for_text_completion=False, return_image_masks=True) .
       这行代码报错原因是：
processor返回的是一个字典{'input_ids': tensor([[    0,  8197,  8711,  ..., 24505, 16582,  8710]]), 'attention_mask': tensor([[1, 1, 1,  ..., 1, 1, 1]]),   'pixel_values': tensor([[[[-0.0484,.....]]]]) } 。不能将字典解包给两个变量。我也尝试过将字典里的 input_ids 键拿出来赋值给inputs，'attention_mask' 键拿出来赋值给sub_image_masks。但是后续又会报维度不齐的错误。
       我已经反复检查了环境配置，也确认完成了 Setup 部分的操作。但是仍然无法解决这个问题，所以想寻求您的帮助。

**Comment by MrGH-boss** [comment]
已经从“requirement”提问中得到解决方法。
